Chat/Controllers/ContosoHandshakeController.cs

```diff
@@ -31,6 +32,16 @@ public ContosoHandshakeController(IChatAdminThreadStore store, IUserTokenManager
 			_userTokenManager = userTokenManager;
 			_chatGatewayUrl = Utils.ExtractApiChatGatewayUrl(chatConfiguration["ResourceConnectionString"]);
 			_resourceConnectionString = chatConfiguration["ResourceConnectionString"];
+
+			if (!store.Store.ContainsKey("acs_ve_06_07_2021"))
+			{
+				var eventInfo = new ACSEvent
+				{
+					sessionThreadIds = new List<string>() { "19:yXlrdWXkXW8LPAXmDrjcpzg-VwcXz8sE14kWXGez2Qs1@thread.v2" },
+					sessionThreadModeratorIds = new List<string>() { "8:acs:85c99b9e-f6e1-408c-90d9-e37b6ad0e7c3_0000000a-8a5c-fc1b-1000-343a0d009850" }
```

Comment:
"8:acs:85c99b9e-f6e1-408c-90d9-e37b6ad0e7c3_0000000a-8a5c-fc1b-1000-343a0d009850" <= MRI that represents a user?

Reply:
yes, specifically the user who initiated the thread creation